source/_sass/page.scss

```diff
@@ -1,333 +1,330 @@
-	}
-	// rotate the lines to form the x shape
-	&:before {
-		-webkit-transform: rotate3d(0,0,1,45deg);
-		transform: rotate3d(0,0,1,45deg);
-	}
-	&:after {
-		-webkit-transform: rotate3d(0,0,1,-45deg);
-		transform: rotate3d(0,0,1,-45deg);
-	}
+  // hide the middle line
+  background: transparent;
+  // overlap the lines by setting both their top values to 0
+  &:before,
+  &:after {
+    -webkit-transform-origin: 50% 50%;
+    transform-origin: 50% 50%;
+    top: 0;
+    width: $button-size;
+  }
+  // rotate the lines to form the x shape
+  &:before {
+    -webkit-transform: rotate3d(0,0,1,45deg);
+    transform: rotate3d(0,0,1,45deg);
+  }
+  &:after {
+    -webkit-transform: rotate3d(0,0,1,-45deg);
+    transform: rotate3d(0,0,1,-45deg);
+  }
 }
 // Style the toggle menu link and hide it
 .nav .navtoogle {
-	@include font-rem(18);
-	font-weight: normal;
-	background-color: $black;
-	color: $white;
-	border: none;
-	cursor: pointer;
-	@media #{$small} {
-		display: none;
-	}
+  @include font-rem(18);
+  font-weight: normal;
+  background-color: $black;
+  color: $white;
+  border: none;
+  cursor: pointer;
+  @media #{$small} {
+    display: none;
+  }
 }
 .nav button {
-	border: none;
-	background: none;
+  border: none;
+  background: none;
 }
 .navtoogle i {
-	z-index:-1;
+  z-index:-1;
 }
 .icon-menu {
-	position: relative;
-	top: 3px;
-	line-height: 0;
+  position: relative;
+  top: 3px;
+  line-height: 0;
 }
 // When JavaScript is disabled, we hide the toggle button
 .no-js .nav .navtoggle {
-	display: none;
+  display: none;
 }
 // When JavaScript is disabled, we show the menu
 .no-js .nav ul {
-	max-height: 30em;
-	overflow: hidden;
+  max-height: 30em;
+  overflow: hidden;
 }
 // When JavaScript is enabled, we hide the menu
 .js .nav ul {
-	max-height: 0;
-	overflow: hidden;
-	@media #{$small} {
-		max-height: 30em;
-	}
+  max-height: 0;
+  overflow: hidden;
+  @media #{$small} {
+    max-height: 30em;
+  }
 }
 // Displaying the menu when the user has clicked on the button
 .js .nav .active + ul {
-	max-height: 30em;
-	overflow: hidden;
-	-webkit-transition: max-height .4s;
-	-moz-transition: max-height .4s;
-	-o-transition: max-height .4s;
-	-ms-transition: max-height .4s;
-	transition: max-height .4s;
+  max-height: 30em;
+  overflow: hidden;
+  -webkit-transition: max-height .4s;
+  -moz-transition: max-height .4s;
+  -o-transition: max-height .4s;
+  -ms-transition: max-height .4s;
+  transition: max-height .4s;
 }
 #main {
-	counter-reset: captions;
-	@include container;
-	@include clearfix;
-	clear: both;
-	margin-top: 2em;
-	h1 {
-		margin-top: 0;
-	}
-	.post,
-	.page {
-		@include container;
-		@include grid(12,10);
-		@include prefix(12,1);
-		@include suffix(12,1);
-		margin-bottom: 2em;
-		@media #{$small} {
-			@include grid(12,6);
-			@include prefix(12,0.5);
-			@include suffix(12,0.5);
-		}
-		@media #{$x-large} {
-			@include grid(12,4.5);
-			@include prefix(12,2);
-		}
-	}
+  counter-reset: captions;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets